src/raygun.js

```diff
@@ -646,6 +646,10 @@ var raygunFactory = function (window, $, Raygun, undefined) {
             }
         }
 
+        if (Raygun.Utilities.isReactNative() && !Raygun.Utilities.contains(options.tags), 'React Native') {
```

Comment:
Due to a misplaced parenthesis this line effectively evaluates to if ('React Native') which is always true.

Should be:

```javascript
if (Raygun.Utilities.isReactNative() && !Raygun.Utilities.contains(options.tags, 'React Native')) {
```